cli/commonsub/initcmd/initcmd.go

```diff
@@ -39,14 +39,14 @@ func KeypairStep(f fsdir.T) error {
 		return err
 	}
 	log.Printf("writing ed25519 private key seed to %s", f.Path(Seed))
-	if err = f.Set(jsonb.B(sk.Seed()), Seed); err != nil {
+	if err = f.SetIndented(jsonb.B(sk.Seed()), Seed); err != nil {
```

Comment:
actually writing keys as "indented" seems funny. it doesn't really make a difference in the output since the value is a string, but i think `Set` would be more appropriate since in this case we don't care about the indentation